.gitignore

```diff
@@ -4,6 +4,9 @@
 
 ### Node ###
 # Logs
+.vscode
+dist
+package-lock.json
```

Comment:
Wondering why you are ignoring package-lock.json, any particular reason why?

Reply:
I started ignoring when someone requested it in a pr, I made yesterday.

Reply:
Okay, please commit this file.